actix/src/fut/stream/fold.rs

```diff
@@ -5,32 +5,32 @@ use futures_core::ready;
 use pin_project_lite::pin_project;
 
 use crate::actor::Actor;
-use crate::fut::{ActorFuture, ActorStream, IntoActorFuture};
+use crate::fut::{ActorFuture, ActorStream};
 
 pin_project! {
     /// Stream for the [`fold`](super::ActorStreamExt::fold) method.
     #[derive(Debug)]
     #[must_use = "streams do nothing unless polled"]
-    pub struct Fold<S, F, Fut, T> {
+    pub struct Fold<S, Fn, Fut, T> {
```

Comment:
honestly, i prefer using F for this because Fn can confuse some parsers

Reply:
Ok would change it to F. My intention is to make the difference between closure and the future returned.
F would also do.

Reply:
my personal preference is F and Fut